generators/entity/files.js

```diff
@@ -213,7 +213,7 @@ const angularFiles = {
             templates: [
                 {
                     file: 'entities/_entity-management.component.html', method: 'copyHtml', template: true,
-                    renameTo: generator => `entities/${generator.entityFolderName}/${generator.entityFileName}.component.html`
+                    renameTo: generator => `entities/${generator.entityFolderName}/${generator.entityPluralFileName}.component.html`
```

Comment:
lets keep it as `entityFileName` and fix the tests so that this is compatible to angular CLI